File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/bike/ReserveBikeResponse.java
package com.lattis.ellipse.data.network.model.response.bike;

import com.google.gson.annotations.SerializedName;
import com.lattis.ellipse.data.network.model.response.AbstractApiResponse;

/**
 * Created by ssd3 on 4/3/17.
 */

public class ReserveBikeResponse extends AbstractApiResponse {

    @SerializedName("payload")
    ReserveBikeDataResponse reserveBikeDataResponse;

    public ReserveBikeDataResponse getReserveBikeResponse() {
        return reserveBikeDataResponse;
    }

    @Override
    public String toString() {
        return "GetUserResponse{" +
                "FindBikeDataResponse=" + reserveBikeDataResponse +
                '}';
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/bike/ReserveBikeResponse.kt
package com.lattis.data.entity.response.bike

import com.google.gson.annotations.SerializedName
import com.lattis.data.entity.response.base.AbstractApiResponse

class ReserveBikeResponse : AbstractApiResponse() {

    @SerializedName("payload")
    var reserveBikeResponse: ReserveBikeDataResponse? = null
        internal set

    override fun toString(): String {
        return "GetUserResponse{" +
                "FindBikeDataResponse=" + reserveBikeResponse +
                '}'.toString()
    }
}
